timesheet_helpdesk_ticket/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Timesheet for Helpdesk's Tickets",
+    'summary': """
+        Añade las funciones de partes de horas a los tickets de Helpdesk Management """,
```

Comment:
Review guidelines and not use Spanish lang

Reply:
Fixed.
'summary': 'Add HR Timesheet to the tickets for Helpdesk Management.',